tidb-cloud/scale-tidb-cluster.md

```diff
@@ -99,19 +99,50 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
 
 For a running cluster, AWS and Google Cloud do not allow in-place storage capacity downgrade.
 
-## Increase node size
+## Change node size
+
+You can change the size (including vCPUs and memory) of TiDB, TiKV, or TiFlash nodes.
+
+### Increase node size
```

Comment:
Is it possible to merge `### Increase node size` and `### Decrease node size`? They look basically the same.

Reply:
The only differences lie in the words `increase` and `decrease`.
<img width="1698" alt="image" src="https://user-images.githubusercontent.com/59654584/223913623-a2c5bc63-d69e-4be6-9ecb-a6ad4ebc4f8e.png">

Reply:
I think we can merge `increase` and `decrease`, @SunRunAway

Reply:
I have no preference.

Reply:
Also, there are "Increase node number" and "Decrease node number"  In "Change node number".  So if changes are to be made, the entire document will hopefully be consistent.

Reply:
"Decrease node number" comes with a warning on _insufficient storage space_. If we're to refactor the doc, please take notice of that warning. Rest LGTM